### ISSUE TITLE
INFO:root:Error processing or transcribing

### ISSUE THREAD

**Issue by handtrixx** [McCloudS/subgen#88]
Hi,
I am using the dockerfile to start subgen and had no issues for months. Thank you so much for creating this great piece of software! Sincerely since ca. 1 week the logs are only showing:
NFO:root:Error processing or transcribing ...: 'True' is not a valid language code (accepted language codes: af, am, ar, as, az, ba, be, bg, bn, bo, br, bs, ca, cs, cy, da, de, el, en, es, et, eu, fa, fi, fo, fr, gl, gu, ha, haw, he, hi, hr, ht, hu, hy, id, is, it, ja, jw, ka, kk, km, kn, ko, la, lb, ln, lo, lt, lv, mg, mi, mk, ml, mn, mr, ms, mt, my, ne, nl, nn, no, oc, pa, pl, ps, pt, ro, ru, sa, sd, si, sk, sl, sn, so, sq, sr, su, sv, sw, ta, te, tg, th, tk, tl, tr, tt, uk, ur, uz, vi, yi, yo, zh, yue)
I do not passe any other environment variables then: NAMESUBLANG=yo and wondering if I am the only one who is experiencing that issue.

**Comment by McCloudS** [comment]
What is your full docker-compose and how are you calling Subgen to run (Bazarr, Plex/other, docs)?

For whatever reason 'True' is being passed as the language code somewhere in your process.

**Comment by handtrixx** [comment]
Thanks a lot for your quick reply. Exactly, I am searching for that reason since I don't pass a "True" anywhere.
I am using docker-compose in linux and I call Subgen by a Jellyfin webhook. As mentioned it was working extremly fine before.
My docker-compose.yml
...
  subgen:
    container_name: subgen
    tty: true
    build:
        context: .
        dockerfile: subgen.Dockerfile
    restart: unless-stopped
    volumes:
         ...
        - ./subgen/models:/subgen/models
    environment:
      - NAMESUBLANG=yo
      - PROCADDEDMEDIA=False
      - JELLYFINSERVER=...
      - JELLYFINTOKEN=...
    networks:
      - default
   ....
Doesn't matter if I directly use your image or do it by dockerfile. but just in case, the dockerfile is:
FROM python:3.11-slim-bullseye

WORKDIR /subgen

ADD https://raw.githubusercontent.com/McCloudS/subgen/main/requirements.txt /subgen/requirements.txt

RUN apt-get update \
    && apt-get install -y \
        python3 \
        python3-pip \
        ffmpeg \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* \
    && pip install -r requirements.txt

ENV PYTHONUNBUFFERED=1

ADD https://raw.githubusercontent.com/McCloudS/subgen/main/launcher.py /subgen/launcher.py
ADD https://raw.githubusercontent.com/McCloudS/subgen/main/subgen.py /subgen/subgen.py

CMD [ "bash", "-c", "python3 -u launcher.py" ]

Am I missing something?

**Comment by McCloudS** [comment]
You're right, I broke this somehow last update.  Thanks for finding it.

If you re-pull the image in about 15 minutes, you're good.  Or you can set `UPDATE=True` and it will download the new script version.

I'll double check the fix when I get home, but it should be it.

**Comment by handtrixx** [comment]
Wow, that was fast. I can confirm this one is fixed. I owe you a beer

**Comment by handtrixx** [comment]
just an additional note, this acutally solved a secondary issue as well. i had some videos that ran through successfully when i I forced the language to a specific one, which I did because the detection wasn't working anymore until you fixed the bug. But then the result was an empty translation (srt) file. Maybe worth to check if forcing the language still causes issues, since I did not have to try that anymore.
